README.md

```diff
@@ -33,3 +33,18 @@ The Raspberry Pi hardware I2C does not support clock stretching. You have two op
 
 You may need to add your user to the i2c group, e.g. `sudo usermod -aG i2c nvidia`. It should just work after that.
 
+# ROS2
```

Comment:
first part of the instructions should be to check out the ros2 branch.

Reply:
I'm wondering should it be a ros2 branch or crystal-devel branch?

Reply:
Good question.  I don't think there's a standard yet, but maybe @clalancette knows more?

Reply:
robot_localization uses crystal-devel, sophus and ecl_core use /release/1.0-crystal, but those are actually released, so maybe we should go with crystal-devel.